test/functional/tests/initialize/test_initialize_load.py

```diff
@@ -0,0 +1,360 @@
+        if cache_mode != cache.get_cache_mode():
+            TestRun.fail("Cache mode are different.")
+        if cache_line_size != cache.get_cache_line_size():
+            TestRun.fail("Cache line size are different.")
+        if cleaning_policy != cache.get_cleaning_policy():
+            TestRun.fail("Cleaning policy are different.")
+        if cleaning_policy == CleaningPolicy.alru:
+            if alru != cache.get_flush_parameters_alru():
+                TestRun.fail("Cleaning policy parameters are different.")
+        if cleaning_policy == CleaningPolicy.acp:
+            if acp != cache.get_flush_parameters_acp():
+                TestRun.fail("Cleaning policy parameters are different.")
+
+    with TestRun.step("Run FIO again on cache's exported object"):
+        for i in range(0, core_number):
+            Fio().create_command() \
+                .io_engine(IoEngine.libaio) \
+                .io_depth(64) \
+                .size(Size(4, Unit.GibiByte)) \
+                .read_write(ReadWrite.randrw) \
+                .block_size(Size(64, Unit.MebiByte)) \
+                .target(f"{core[i].system_path}") \
+                .run()
+
+    with TestRun.step("Stop caches."):
+        casadm.stop_all_caches()
+
+
+@pytest.mark.parametrize("core_number", [1, 4])
+@pytest.mark.parametrize("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrize("cache_mode", [[CacheMode.WT, CacheMode.WB],
+                                        [CacheMode.WB, CacheMode.WA],
+                                        [CacheMode.WA, CacheMode.PT],
+                                        [CacheMode.PT, CacheMode.WO],
+                                        [CacheMode.WO, CacheMode.WT]])
+@pytest.mark.parametrize("cache_line_size", [[CacheLineSize.LINE_4KiB, CacheLineSize.LINE_8KiB],
+                                             [CacheLineSize.LINE_8KiB, CacheLineSize.LINE_16KiB],
+                                             [CacheLineSize.LINE_16KiB, CacheLineSize.LINE_32KiB],
+                                             [CacheLineSize.LINE_32KiB, CacheLineSize.LINE_64KiB],
+                                             [CacheLineSize.LINE_64KiB, CacheLineSize.LINE_4KiB]])
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_x_to_one_diff_params(cache_mode, cleaning_policy, cache_line_size, core_number):
+    f"""
+        title: Negative initialize test for loading CAS with 1 cache and {core_number} cores.
+        description: |
+          Verify that loading cache configurations works properly in every mode
+          with 1 cache and {core_number} cores. Use different parameters with load command.
+        pass_criteria:
+          - All test steps complete without errors.
+          - OpenCAS should load with saved configuration.
+          - No data corruption is found.
```

Comment:
Is it being verified at all?

Reply:
Now I added checking for errors in cache.